config/katello.migrations/220411132123-rename-rex-ssh-rex-script.rb

```diff
@@ -0,0 +1,4 @@
+unless answers['foreman_proxy::plugin::remote_execution::ssh'].nil?
```

Comment:
Any reason you have a different migration here than the others?

Reply:
It's like ordering ice cream but getting two flavors, its still ice cream. j/k total copy/paste failure.